### PR TITLE
Update plugins to use our helper merge function

### PR DESCRIPTION
**Pull request by mszostok**
## Description

Changes proposed in this pull request:

- Use `pluginx` funcs

## Related issue(s)

- https://github.com/kubeshop/botkube/issues/925

## Note
I didn't use my fork, I will do that for next PRs.

### CODE REVIEW

**Comment by mszostok** [comment]
I forgot to update the README.md, I will address that after first review.

File: go.mod [hunk 1]
```diff
@@ -8,13 +8,32 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 )
 
+// Remove after merging: https://github.com/kubeshop/botkube/pull/951
+replace github.com/kubeshop/botkube v0.17.0 => github.com/mszostok/botkube v0.12.5-0.20230124234207-f9070a8fbdaa
```
Comment (pkosiec) [inline go.mod:12]: Please see the review here: https://github.com/kubeshop/botkube/pull/951/files

**Review by pkosiec — COMMENTED** [review]
Code looks good, I'll test it along with reading the blog post and post a final approval 👍

File: .goreleaser.yaml [hunk 2]
```diff
@@ -29,6 +29,16 @@ builds:
     goarch: *goarch
```
Comment (pkosiec) [inline .goreleaser.yaml:29]: BTW If we merge this here, then someone following the blog post will get the gh plugin out of the box, right? so maybe we should change the name in the blogpost to something different?

Comment (pkosiec) [inline .goreleaser.yaml:29]: PTAL at this comment: https://github.com/kubeshop/botkube-docs/pull/204#discussion_r1087549644

**Review by pkosiec — APPROVED** [review]
Works really good!

**Pull request merged**